nextcloudappstore/core/models.py

```diff
@@ -88,34 +89,52 @@ def can_update(self, user: User) -> bool:
     def can_delete(self, user: User) -> bool:
         return self.owner == user
 
-    def releases_by_platform_v(self):
+    def releases_by_platform_v(self, nightlies=False):
```

Comment:
Ah, it only returns nightlies. In that case I'd opt for a separate method, e.g.: nighly_releases_by_platform, otherwise its super confusing :)